hibernate-core/hibernate-core.gradle

```diff
@@ -70,8 +70,8 @@ dependencies {
     testRuntimeOnly libs.byteBuddy
     testRuntimeOnly testLibs.weld
     testRuntimeOnly testLibs.wildFlyTxnClient
-    testRuntimeOnly jakartaLibs.jsonb
-    testRuntimeOnly libs.jackson
+    testImplementation jakartaLibs.jsonb
```

Comment:
might be good to move these dependencies next to the rest that are declared as `testImplementation` after L68.

Reply:
I don't think so, we should minimize the visibility of dependencies.

Reply:
I am not sure I understand your point. 

They are sorted in the build file exactly because of better visibility, and it keeps the gradle build file nice and tidy.

Reply:
`jacksonXml` and `jacksonJsr310` is not required by sources, keep them as `runtimeOnly` to avoid potential classpath pollution.